css-three-column-layout/styles.css

```diff
@@ -0,0 +1,64 @@
+* {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  src:
+    url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2")
+    format("woff2");
```

Comment:
This can also be removed.

Reply:
Thanks Cody. I just removed it at re-pushed. I will make to only keep the font family and when needed, font-weight.

Thanks again!